src/NumberVisuals/index.tsx

```diff
@@ -96,10 +89,10 @@ function NumberVisuals({
   const toggleHover = () => setOnHover(!onHover);
   return (
     <div className={classes.root}>
-      <Typography variant="h5">{subtitle}</Typography>
+      <Typography className={classes.h6}>{subtitle}</Typography>
```

Comment:
I would say call the class `classes.subtitle `

Same for the rest, give theme more descriptive names.

Reply:
@karimkawambwa Done